libraries/AP_Gripper/AP_Gripper_Servo.cpp

```diff
@@ -57,36 +49,34 @@ bool AP_Gripper_Servo::has_state_pwm(const uint16_t pwm) const
         // servo still moving....
         return false;
     }
-#if CONFIG_HAL_BOARD == HAL_BOARD_SITL
-    if (is_releasing) {
-        gcs().send_text(MAV_SEVERITY_INFO, "Gripper load released");
-    } else {
-        gcs().send_text(MAV_SEVERITY_INFO, "Gripper load grabbed");
-    }
-#endif
     return true;
 }
 
-
 bool AP_Gripper_Servo::released() const
 {
-    return has_state_pwm(config.release_pwm);
+    return (config.state == AP_Gripper::STATE_RELEASED);
 }
 
 bool AP_Gripper_Servo::grabbed() const
 {
-    return has_state_pwm(config.grab_pwm);
+    return (config.state == AP_Gripper::STATE_GRABBED);
```

Comment:
Looks like these two are the only users of `has_state_pwm` So you should be able to remove the whole function.

Reply:
Ignore me, I now see there moved to the `update_gripper` function.